docs/LOCALIZATION.md

```diff
@@ -0,0 +1,248 @@
+      -  The language code used in Web Chat only indicate the written language.
+      -  Thus, when performing text-to-speech, the language code will be remapped for oral language.
+      -  For example, written language in Hong Kong is `zh-Hant-HK`, while the oral language is `zh-HK` (Cantonese). And the written language in Taiwan is `zh-Hant`, while the oral language is `zh-TW` (Taiwanese Mandarin).
+   -  (Optional) `COGNITIVE_SERVICES_*` is used to indicate whether the language supported Cognitive Services Speech Services.
+      -  `COGNITIVE_SERVICES_SPEECH_TO_TEXT` is used to indicate the language is supported by Cognitive Services Speech-to-Text Service.
+      -  `COGNITIVE_SERVICES_TEXT_TO_SPEECH` is used to indicate the language is supported by Cognitive Services Text-to-Speech Service. If the service support neural voices, set it to `"neural"`, otherwise, `true`
+      -  https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support
+-  Update `/packages/component/src/Utils/normalizeLanguage.js` and add a normalization logic for ISO language.
+   -  This is for cleaning up data provided by developers.
+   -  For example, if the language starts with `ja`, it will be mapped to `ja-JP`.
+-  Add a new language to `/packages/component/src/Localization/`.
+   -  Copy `en-US.json` as the base template.
+
+## Updating strings of an existing language
+
+There are two types of supported languages:
+
+-  Maintained by Microsoft
+-  Contributed by the community
+
+For strings that are maintained by Microsoft, please file a bug to us. It will take us time as we need perform additional checks to maintain the quality.
+
+For strings that are contributed by the community, please submit a pull request to us.
+
+## Adding new strings for new UI
+
+To add new strings to be used in the UI, please update `/packages/component/src/localization/en-US.json`. Our localization team will pick up the new strings and translate them to all other officially supported languages.
+
+## Overriding localization strings
+
+Developers can override localization strings in one of two ways. An `overrideLocalizedStrings` prop will be passed and it can either be a plain object or a function.
+
+```js
+window.WebChat.renderWebChat(
+   {
+      locale: 'en-US',
+
+      overrideLocalizedStrings: (strings, language) => ({
+         ...strings,
+         TEXT_INPUT_PLACEHOLDER: 'What is on your mind?'
+      }),
+
+      overrideLocalizedStrings: {
+         TEXT_INPUT_PLACEHOLDER: 'What is on your mind?'
+      }
+   },
+   document.getElementById('webchat')
+);
+```
+
+## Design
+
+### User stories
+
+The localization feature is backed by these user stories:
+
+-  Translated by Microsoft localization team with automated pipeline
+-  Overrideable by developers
+   -  Change the "Type your message" placeholder text
+
```

Comment:
Is this necessary? I don't think it's relevant to the customer.

Reply:
If it is, I think it needs more clarity

Reply:
Updated.